service/matching/matchingEngine.go

```diff
@@ -215,13 +228,12 @@ func (e *matchingEngineImpl) AddDecisionTask(
 	domainID := request.GetDomainUUID()
 	taskListName := request.TaskList.GetName()
 	taskListKind := request.TaskList.Kind
-
-	e.logger.Debug(
-		fmt.Sprintf("Received AddDecisionTask for taskList=%v, WorkflowID=%v, RunID=%v, ScheduleToStartTimeout=%v",
-			request.TaskList.GetName(),
-			request.Execution.GetWorkflowID(),
-			request.Execution.GetRunID(),
-			request.GetScheduleToStartTimeoutSeconds()))
+	e.logger.Debug("Received AddDecisionTask",
+		tag.WorkflowTaskListName(request.TaskList.GetName()),
```

Comment:
I have seen issues with sticky task list before - may consider logging task list type and/or sticky task list name vs base task list name.

Reply:
good idea, will add this. Thanks

Reply:
I think it will be helpful to also log the task scheduledID, which can help us identify a certain decision task in the workflow. Otherwise if the workflow is long and have many decision task, it's hard to find the log for one particular decision task. Same suggestion for the AddActivityTask log. 

I am actually making the same change for those log messages when improving the visibility into transfer task processing logic as I need some messaged logged at Info level. :) I will resolve the conflicts once you land the diff.